app/editor/src/features/content/form/MediaSummary.tsx

```diff
@@ -14,6 +14,7 @@ export interface IMediaSummaryProps {
   stream: any;
   contentType?: ContentTypeName;
   setShowExpandModal?: (show: boolean) => void;
+  product: string;
```

Comment:
Change component property to `isSummaryRequired: boolean`.  Default the value to `true`.

Reply:
Updated, thanks